### PR TITLE
cloud: update kafka changefeed doc based on docs usability testing

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by cchenkey — APPROVED** [review]

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 1]
```diff
@@ -77,22 +77,26 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
...
+
+    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs.
 
    - **Distribute changelogs by table to Kafka Topics**
 
-        If you want the changefeed to create a dedicated Kafka topic for each table, choose this mode. Then, all Kafka messages of a table are sent to a dedicated Kafka topic. You can define topic names for tables by setting the topic prefix, separator, and suffix. For example, if you set the separator as `_`, the topic names are in the format of `<Prefix>_<DatabaseName>_<TableName>_<Suffix>`.
+        If you want the changefeed to create a dedicated Kafka topic for each table, choose this mode. Then, all Kafka messages of a table are sent to a dedicated Kafka topic. You can customize topic names for tables by setting a topic prefix, a separator between a database name and table name, and a suffix. For example, if you set the separator as `-`, the topic names are in the format of `<Prefix><DatabaseName>-<TableName><Suffix>`.
```
Comment (leoppro) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:91]:
```suggestion
        If you want the changefeed to create a dedicated Kafka topic for each table, choose this mode. Then, all Kafka messages of a table are sent to a dedicated Kafka topic. You can customize topic names for tables by setting a topic prefix, a separator between a database name and table name, and a suffix. For example, if you set the separator as `_`, the topic names are in the format of `<Prefix><DatabaseName>_<TableName><Suffix>`.
```

**Review by leoppro — APPROVED** [review]
LGTM

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 2]
```diff
@@ -77,22 +77,26 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
...
+    - In the **Schema Registry** area, fill in your schema registry endpoint. If you enable **HTTP Authentication**, the fields for user name and password are displayed and automatically filled in with your TiDB cluster endpoint and password.
 
-4. In the **Topic Distribution** area, select a distribution mode, specify a default Kafka topic name, and set the corresponding topic prefix, separator, and suffix.
+4. In the **Topic Distribution** area, select a distribution mode, and then fill in the topic name configurations according to the mode.
 
-    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs. If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
+    If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
+
+    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:87]:
```suggestion
    The distribution mode controls how the changefeed creates Kafka topics, by table, by database, or creating one topic for all changelogs.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 3]
```diff
@@ -77,22 +77,26 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
...
+    - In the **Decimal** and **Unsigned BigInt** configurations, specify how TiDB Cloud handles the decimal and unsigned bigint data types in Kafka messages.
+    - In the **Schema Registry** area, fill in your schema registry endpoint. If you enable **HTTP Authentication**, the fields for user name and password are displayed and automatically filled in with your TiDB cluster endpoint and password.
 
-4. In the **Topic Distribution** area, select a distribution mode, specify a default Kafka topic name, and set the corresponding topic prefix, separator, and suffix.
+4. In the **Topic Distribution** area, select a distribution mode, and then fill in the topic name configurations according to the mode.
 
-    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs. If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
+    If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:85]:
```suggestion
    If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribution Mode** drop-down list.
```

**Pull request merged**